frontend/public/components/namespace.jsx

```diff
@@ -1111,7 +1111,7 @@ export const NamespaceDetails = ({ obj: ns, customData }) => {
 
 const RolesPage = ({ obj: { metadata } }) => (
   <RoleBindingsPage
-    createPath={`/k8s/ns/${metadata.name}/rolebindings/~new?rolekind=Role`}
+    createPath={`/k8s/ns/${metadata.name}/rolebindings/~new?rolekind=ClusterRole`}
```

Comment:
Hey, @cyril-ui-developer. I'm unclear why this change was needed.

Reply:
> Hey, @cyril-ui-developer. I'm unclear why this change was needed.

@spadgett The objects in `sortedList` which the role dropdown make use of are concatenated with `xxxx-ClusterRole`. So if `kindLabel` is  set to `Role` there is not match. That was my thought for changed.
https://github.com/openshift/console/blob/master/frontend/public/components/utils/list-dropdown.jsx#L110
Also, the role dropdown items are from `/clusterroles` endpoint. 

However, I investigated further, and found out that switching OR operator operands fix the issue. 
https://github.com/openshift/console/blob/master/frontend/public/components/RBAC/bindings.jsx#L645
` selectedKeyKind={roleRef.kind || _.get(fixed, 'roleRef.kind')}`
Because the `roleRef.kind = "ClusterRole"` and `_.get(fixed, 'roleRef.kind') = "Role"` if the context is from Project.  


I think the `roleRef.kind` and  `fixed.roleRef.kind` should be the same value or one is present and the other undefined. 
If the context is from User Mgt,  the `fixed.roleRef.kind` is `null`

Reply:
Hey, @cyril-ui-developer. Sorry for not responding sooner.

I think the mistake is that we're passing a `roleKind` at all? If I'm reading the code right, `roleKind` should only be used with a specific role name. We should be able to handle both cluster roles and namespaced roles here, and we should make sure both work.

Reply:
> Hey, @cyril-ui-developer. Sorry for not responding sooner.
> 
> I think the mistake is that we're passing a `roleKind` at all? If I'm reading the code right, `roleKind` should only be used with a specific role name. We should be able to handle both cluster roles and namespaced roles here, and we should make sure both work.

@spadgett No worries. 

If I got you right , the `roleKind` should be pass if the context is from Roles page only. 

So we need to remove the `roleKind` in the  namespace url query to fix the issue.

Reply:
I believe that's right.

Reply:
> I believe that's right.

@spadgett Updated. PTAL.